_data/packages.yml

```diff
@@ -532,7 +569,7 @@
     - name: Hassan Kibirige
       github_username: has2k1
   archive: '[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10776842.svg)](https://doi.org/10.5281/zenodo.10776842)'
-  version_accepted: 0.11.0
+  version_accepted: '[0.11.0](https://github.com/vnmabus/rdata/releases/tag/0.11.0)'
```

Comment:
Maybe for this we want two fields

version_accepted can just be a numeric value (not a list)
version_accepted_archive can be the github link to a release. 
archive can refer to zenodo doi archive (and we can add a comment to our issue template about this)
joss: joss doi